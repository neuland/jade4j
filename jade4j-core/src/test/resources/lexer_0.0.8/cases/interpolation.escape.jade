
- var id = 42;
foo
  | some
  | \#{text}
  | here
  | My ID #{"is {" + id + "}"}