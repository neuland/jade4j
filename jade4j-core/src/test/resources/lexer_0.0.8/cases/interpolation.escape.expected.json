{"type":"newline","line":2}
{"type":"code","line":2,"val":"var id = 42;","escape":false,"buffer":false}
{"type":"newline","line":3}
{"type":"tag","line":3,"val":"foo","selfClosing":false}
{"type":"indent","line":4,"val":2}
{"type":"text","line":4,"val":"some"}
{"type":"newline","line":5}
{"type":"text","line":5,"val":"\\#{text}"}
{"type":"newline","line":6}
{"type":"text","line":6,"val":"here"}
{"type":"newline","line":7}
{"type":"text","line":7,"val":"My ID #{\"is {\" + id + \"}\"}"}
{"type":"outdent","line":7}
{"type":"eos","line":7}